demo/compose.rviz.yaml

```diff
@@ -14,8 +18,6 @@ services:
     environment:
       - DISPLAY=${DISPLAY:?Source of setup_virtual_desktop.sh required!}
       - LIBGL_ALWAYS_SOFTWARE=1
```

Comment:
to make it more standard this envs can be moved to `cpu-config` like [here](https://github.com/husarion/rosbot-xl-autonomy/blob/main/compose.pc.yaml)